index.html

```diff
@@ -1,13 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>AuxNudge</title>
-    <link rel="stylesheet" href="./assets/css/style.css">
-</head>
-<body>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body>
+    <!-- Header -->
+    <section class="hero is-medium is-link">
+      <div class="hero-body">
+        <p class="title">Aux Nudge</p>
+        <p class="subtitle">Search for songs you want to play...</p>
+      </div>
+    </section>
+    <!-- Directions for Website Use -->
+    <div class="container is-fluid">
+      <div class="notification">
+        This container is <strong>fluid</strong>: it will have a 32px gap on
+        either side, on any viewport size.
+      </div>
+    </div>
+    <div class="container is-fluid">
+      <div class="columns">
+        <!-- Search For Area -->
+        <div class="column is-one-third">
+          <div class="field has-addons">
+            <label class="label"></label>
+            <div class="control">
+              <input class="input" type="text" placeholder="Enter Word" />
+            </div>
+          </div>
```

Comment:
It looks like this `</div>` tag is closing the field class div before the search button. If the closing tag is moved after the search button this may fix the formatting issue we were having.